core/app/[locale]/(default)/(auth)/login/page.tsx

```diff
@@ -28,20 +36,26 @@ export default async function Login() {
         submitLabel={t('Form.logIn')}
         title={t('heading')}
       >
-        <div className="">
-          <h3 className="mb-3 text-xl font-bold lg:text-2xl">{t('CreateAccount.heading')}</h3>
-          <p className="text-base font-semibold">{t('CreateAccount.accountBenefits')}</p>
-          <ul className="mb-4 list-disc ps-4">
-            <li>{t('CreateAccount.fastCheckout')}</li>
-            <li>{t('CreateAccount.multipleAddresses')}</li>
-            <li>{t('CreateAccount.ordersHistory')}</li>
-            <li>{t('CreateAccount.ordersTracking')}</li>
-            <li>{t('CreateAccount.wishlists')}</li>
-          </ul>
-          <ButtonLink href="/register" variant="secondary">
-            {t('CreateAccount.createLink')}
-          </ButtonLink>
-        </div>
+        <Slot
+          fallback={
+            <div className="">
+              <h3 className="mb-3 text-xl font-bold lg:text-2xl">{t('CreateAccount.heading')}</h3>
+              <p className="text-base font-semibold">{t('CreateAccount.accountBenefits')}</p>
+              <ul className="mb-4 list-disc ps-4">
+                <li>{t('CreateAccount.fastCheckout')}</li>
+                <li>{t('CreateAccount.multipleAddresses')}</li>
+                <li>{t('CreateAccount.ordersHistory')}</li>
+                <li>{t('CreateAccount.ordersTracking')}</li>
+                <li>{t('CreateAccount.wishlists')}</li>
+              </ul>
+              <ButtonLink href="/register" variant="secondary">
+                {t('CreateAccount.createLink')}
+              </ButtonLink>
+            </div>
+          }
+          label="Login sidebar content"
```

Comment:
Do we need to translate this `label`? Same with the cart page?

Reply:
Hm, it's mainly for the publish dialog and builder. I don't think we're translating these labels anywhere else.